packages/main/test/specs/Calendar.spec.js

```diff
@@ -423,4 +421,14 @@ describe("Calendar general interaction", () => {
 
 		assert.strictEqual(await dayPickerRoot.hasClass("ui5-dp-twocalendartypes"), false, "Secondary Calendar class is applied correctly");
 	});
+
+	it("Focus goes into the current day item of the day picker", async () => {
```

Comment:
I think something like - Focus goes into the **custom selected day** item of the day picker, or **predefined day** or something of that sort would be better name for the test, since for me **Current day** sounds like '**Today**'

Reply:
Ye, I will go with selected, becuase it can be changed at any moment, it's not only predefined.